Packs/IBMResilientSystems/Integrations/IBMResilientSystems/IBMResilientSystems.py

```diff
@@ -27,7 +27,7 @@
 USERNAME = demisto.params()['credentials']['identifier']
 PASSWORD = demisto.params()['credentials']['password']
 USE_SSL = not demisto.params().get('insecure', False)
-
+FETCH_TIME = demisto.params().get('fetch_time', '')
```

Comment:
Maybe add a validation to the user input?

Reply:
verified in the test method